articles/azure-monitor/app/managed-workspaces.md

```diff
@@ -45,6 +45,14 @@ Managed workspaces created by Application Insights follow specific naming conven
 
 You can identify the managing resource by checking the **Managed By** property in the Azure portal.
 
+## Removing managed workspaces
+
+Removing a managed workspace requires that it is not longer connected to any other resoruces or resource groups.  If you want to delete a manage workspace, you must do the following:
+- **Remove the connected workspace-based Application Insights resource**: This can be done by connecting the Application Insights resource to a different Log Analytics workspace or deleting the Application Insights resource.
```

Comment:
```suggestion
1. **Remove the connected workspace-based Application Insights resource**: This can be done by connecting the Application Insights resource to a different Log Analytics workspace or deleting the Application Insights resource.
```